client/src/document/toolbar/edit-actions.tsx

```diff
@@ -91,12 +86,15 @@ export function EditActions({
 
       {!isReadOnly && (
         <li>
-          <Link
-            to={location.pathname.replace("/docs/", "/_edit/")}
+          <a
+            href={github_url.replace("/blob/", "/edit/")}
             className="button"
+            title={`You're going to need to sign in to GitHub first (Opens in a new tab)`}
```

Comment:
Nit

```suggestion
            title="Edit on GitHub. Opens in a new tab"
```

Reply:
I took it from here:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/495429/165310990-33f2265a-a07d-42f2-9c68-d685f052985c.png">

But maybe we don't need a `title` here at all? Since they're running yarn locally, they most likely have a GitHub account already, and opening in a new tab doesn't need to be advertised.

Reply:
Good point, thank you, I have removed the `_blank` target, because there is no reason to enforce this.

Reply:
Good point, thank you, I have removed the `_blank` target, because there is no reason to enforce this.